api/spitsgids/SpitsgidsController.php

```diff
@@ -0,0 +1,135 @@
+<?php
+/* Copyright (C) 2011 by iRail vzw/asbl
+   * © 2015 by Open Knowledge Belgium vzw/asbl
+   *
+   * This class handles the data transfer from Spitsgids tables.
+   *
+   * @author Stan Callewaert
+   */
+
+use MongoDB\Collection;
+
+class SpitsgidsController
```

Comment:
A Controller in a webapp is something that takes care of a HTTP call after that it has been processed by a router... This is not a controller, no?